### DISCUSSION TITLE
bug: Value: Error: not implemented

### DISCUSSION THREAD

**Discussion by SheppardX** [ionic-team/capacitor#3800] [Q&A] [answered]
### Capacitor Version

```
💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 2.4.2
  @capacitor/core: 2.4.2
  @capacitor/android: 2.4.2
  @capacitor/electron: 2.4.2
  @capacitor/ios: 2.4.2

Installed Dependencies:

  @capacitor/cli 2.4.2
  @capacitor/android 2.4.2
  @capacitor/ios 2.4.2
  @capacitor/core 2.4.2
  @capacitor/electron not installed

[success] Android looking great! 👌
  Found 8 Capacitor plugins for ios:
    cordova-plugin-browsertab (0.2.0)
    cordova-plugin-compat (1.2.0)
    cordova-plugin-deeplinks (1.1.1)
    cordova-plugin-inappbrowser (4.0.0)
    cordova-plugin-ionic (5.4.7)
    cordova-plugin-ionic-webview (5.0.0)
    cordova-plugin-whitelist (1.3.4)
    ionic-plugin-deeplinks (1.0.20)
[success] iOS looking great! 👌

```

### Platform(s)
Android

### Current Behavior
we are using angular 9.1.2 and when i starting the app on my note 10 + installed via android studio i get directly the error
![image](https://user-images.githubusercontent.com/6494427/98947126-1c944a80-24f5-11eb-98ac-978eb1b0ac2b.png)

### Expected Behavior
start without error

`npm --version` output: 6.14.8

`node --version` output: v12.11.0

IOS Workes perfectly fine

**Comment by jcesarmobile** [comment, marked as answer]
That means `Keyboard.setAccesoryBarVisible` is not implemented for Android because it's an iPhone only thing and the plugin rejects with unimplemented error, you should put the plugin code inside a try catch.

**Reply by SheppardX** [reply]
Thx, i was totally blind. Now i see what u mean and what the error means !
